src/geotop/matrixview.h

```diff
@@ -0,0 +1,35 @@
+//
+// Created by elisa on 2018/07/27.
+//
+
+#ifndef GEOTOP_MATRIXVIEW_H
+#define GEOTOP_MATRIXVIEW_H
+
+#include "geotop_asserts.h"
+#include "rowview.h"
+
+template <class T> class MatrixView {
+public:
+    T *elem;
+    std::size_t nrh;
+    std::size_t nrl;
+    std::size_t nch;
+    std::size_t ncl;
+
+//    RowView<T> operator[](const std::size_t i) noexcept {
```

Comment:
why are these functions commented?

Reply:
I commented these lines to see if they were used in some source files, since I didn't want it (I wanted to have only ```()```).  And in fact the problem was that, given  
- in struct.geotop.h: ```std::unique_ptr<Tensor<double>> pa``` 
- in pedo.func.cc:  ```MatrixView<double> &&pa``` (passed as argument in a function)
I forgot to change some expression like ```pa[ja][l]``` into ```p(ja,l)``` so I guessed some uninitialized values were used for the calculations.

Anyway I thought that the ```operator[]``` was used by ```operator()``` but it seemed not, since the code was compiling and running.  __Do you know why?__